### ISSUE TITLE
Unable to use AWS presigned urls with agent

### ISSUE THREAD

**Issue by jmartin4563** [newrelic/node-newrelic-aws-sdk#176]
## Description
When using the latest version of our agent, presigned urls can be generated, but cannot be used. When using the repro app, if you disable the agent, you are able to use a presigned url just fine. With the agent, you'll get an error that looks like this:
```
<Code>SignatureDoesNotMatch</Code>
<Message>The request signature we calculated does not match the signature you provided. Check your key and signing method.</Message>
...
</Error>

```

## Expected Behavior
Presigned URLs are usable regardless of whether or not the agent is loaded

## Steps to Reproduce
https://github.com/jmartin4563/aws-sdk-bug

## Additional context
Originally reported in https://github.com/newrelic/node-newrelic/issues/1549#issuecomment-1485054979
This is caused by the changes we made for https://github.com/newrelic/node-newrelic-aws-sdk/issues/168, where we used to use a Symbol, which was transparent to the SDK, but due to changes in aws-sdk and a maintainer's suggestion, we switched to using a string header.

**Comment by bizob2828** [comment]
@pawanakhilMM this was addressed in [v5.0.4](https://github.com/newrelic/node-newrelic-aws-sdk/releases/tag/v5.0.4) of `@newrelic/aws-sdk`, has this addressed your issue?

**Comment by pawanakhilMM** [comment]
Yes @bizob2828 , the issue got resolved. Thanks
